src/services/api.ts

```diff
@@ -94,7 +97,18 @@ export class Api {
         return this.apiClient.get(`/users/${userId}`)
     }
 
-    async getGroups(): Promise<any> {
+    async createGroup(
+        group: Group,
+    ): Promise<NewGroupResponse> {
+        const { name, password } = group
+        return this.apiClient.post('/groups', {
+            name,
+            password
```

Comment:
no need to destructure here, you can pass the `group` directly to the `post` method